editor/editor.css

```
@@ -131,7 +131,15 @@
     height: 100%;
 }
 
-.side-panel .side-panel-button {
+.side-subpanel {
+    padding-top: 5px;
+    background-color: #fff;
+    border-bottom: gray solid 1px;
+    border-top: gray solid 1px;
```

We can use the `lightgrey` color, consistent with most borders in the editor.

```suggestion
    border-bottom: lightgray solid 1px;
    border-top: lightgray solid 1px;
```